CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
-# Contribution guide
+# Contribution Guide
```

Comment:
Why suddelny converting everything to Title Case, in contrast to what we teach?
Same in the `README.md`.

Reply:
In the style guide, we define that we are using Title Case. These were just a few places where we still had sentence case.

> in contrast to what we teach?

Not sure I understand what you mean here.

Reply:
SSE: https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/04_documentation/technical_writing_slides.md#style-guides-44

And since we often refer to preCICE as good practice: https://precice.org/docs-meta-content-guidelines.html#language--style

Maybe reconsider the style guide of the lecture? Where is this defined? I don't see something in the `CONTRIBUTING.md`.

Reply:
> SSE: https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/04_documentation/technical_writing_slides.md#style-guides-44

This is the content of the Google style guide, not of the lecture material.

The one of the lecture is https://github.com/Simulation-Software-Engineering/Lecture-Material/blob/main/docs/styleguide.md

I would not want to change such fundamental things now :grin: 

But good point that we should probably link from the contribution guidelines.

Reply:
So, we are part of the reason Title Case is still a thing in the world... :see_no_evil:

Reply:
shouldn't it? I think it looks more natural on slides. But it's mainly about consistency here, no?

Reply:
I remember we discussed a lot about this when we set the guideline for the preCICE documentation. Title Case:

- was invented as a solution and made sense in print, when it was not easy to distinguish between titles and other elements. We use markdown here, we have ways to signify titles and headings.
- is very confusing to write: what should be capitalized, what not? There are style guides, but in practice they are inconsistent and confusing
- mixes content and style: once you decide to switch how the content should look like, you need to rewrite it.

It was not my original intention to suggest a change in the style guide, I was just suprised and I find it very confusing and inconsistent with everything else we do outside this material.

Reply:
Just discussed with @MakisH: Potential future topic, but let's not do it here.

Reply:
The styleguide is already mention in the contribution guidelines. Resolving here.